shared/src/business/entities/User.ts

```diff
@@ -118,8 +133,17 @@ export class User extends JoiValidationEntity {
     }).messages({ '*': 'Enter state' }),
   };
 
+  static CONTACT_SCHEMA = joi.when('role', {
+    is: joi.invalid(ROLES.judge, ROLES.legacyJudge),
+    otherwise: joi
+      .object()
+      .keys({ phone: User.USER_CONTACT_VALIDATION_RULES.phone })
+      .optional(),
+    then: joi.object().keys(User.USER_CONTACT_VALIDATION_RULES).optional(),
+  });
+
```

Comment:
I would prefer to use `joi.valid`. That said, for reasons I do not fully understand, the approximately six trillion different ways I tried `joi.valid` did not work properly for me, so here we are.